fuzz.go

```diff
@@ -503,31 +504,65 @@ type int63nPicker interface {
 	Int63n(int64) int64
 }
 
-type charRange struct {
-	first, last rune
+// UnicodeRange is a structure that provides the developer with setting
+// one encoding range.
+type UnicodeRange struct {
+	First, Last rune
 }
 
+// UnicodeRanges is a slice that provides the developer with setting
+// the different range of Unicode
+type UnicodeRanges []UnicodeRange
+
 // choose returns a random unicode character from the given range, using the
 // given randomness source.
-func (cr charRange) choose(r int63nPicker) rune {
-	count := int64(cr.last - cr.first + 1)
-	return cr.first + rune(r.Int63n(count))
+func (cr UnicodeRange) choose(r int63nPicker) rune {
+	count := int64(cr.Last - cr.First + 1)
+	return cr.First + rune(r.Int63n(count))
+}
+
+// CustomStringFuzzFunc constructs a function which randomizes strings
```

Comment:
rewrite

Reply:
I just write down a private function of UnicodeRange, this will be panic if Lasr < First.

And the logic of UnicodeRanges is to call the check function of each range.